sel4-extract/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+# set minimum cmake verision
+cmake_minimum_required(VERSION 3.0.0)
+
+# set project name
+project(sel4-extract)
+
+# add executable
+add_executable (sel4-extract sel4-extract.c)
+
+# set compiler options
+set(FLAGS_WAR_AS_ERR -Wall -Wpedantic -Werror)
+# set(FLAGS_MEM_DBG -fsanitize=address -fsanitize=leak -fsanitize=bounds)
+
+target_compile_options(sel4-extract PRIVATE ${FLAGS_WAR_AS_ERR} ${FLAGS_MEM_DBG})
+target_link_options(sel4-extract  PRIVATE ${FLAGS_WAR_AS_ERR} ${FLAGS_MEM_DBG})
+
+# target install
+install(TARGETS sel4-extract RUNTIME DESTINATION bin)
```

Do not use handwritten paths, use [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html#module:GNUInstallDirs). That way it is guaranteed that the paths are being handled correctly (i.e. prefixes etc). This way cross compilation and integration to build systems is easier.

So use:
 ```suggestion
include(GNUInstallDirs)
 
install(TARGETS sel4-extract RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

---

Fixed.